spec/controllers/idv/capture_doc_status_controller_spec.rb

```diff
@@ -33,7 +33,6 @@
         get :show
 
         expect(response.status).to eq(401)
-        expect(response.body).to eq('Unauthorized')
```

Comment:
The text of the body wasn't ever used for anything. We don't even stub it in the JavaScript tests:

https://github.com/18F/identity-idp/blob/dadf6f9e8cdda05522000250ebd6bcd632c710df/spec/javascripts/packages/document-capture-polling/index-spec.js#L82